srcpkgs/libiio/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'libiio'
+pkgname=libiio
+version=0.23
+revision=1
+build_style=cmake
+configure_args="-DUDEV_RULES_INSTALL_DIR=/usr/lib/udev/rules.d -DWITH_SERIAL_BACKEND=YES
+ $(vopt_bool avahi HAVE_DNS_SD) $(vopt_bool doc WITH_DOC) $(vopt_bool python PYTHON_BINDINGS)"
+hostmakedepends="bison flex $(vopt_if doc doxygen)
+ $(vopt_if python 'python3 python3-Sphinx python3-sphinx_rtd_theme')"
+makedepends="libaio-devel libserialport-devel libusb-devel libxml2-devel
+ $(vopt_if avahi avahi-libs-devel)"
+short_desc="Library for interfacing with local and remote Linux IIO devices"
+maintainer="Evgeny Ermakov <evgeny.v.ermakov@gmail.com>"
+license="LGPL-2.1-or-later"
+homepage="https://analogdevicesinc.github.io/libiio"
+distfiles="https://github.com/analogdevicesinc/libiio/archive/v${version}.tar.gz"
+checksum=df1577740ea93dca732e7d7c04786600cb083117901dee793f3d38980754af00
+
+build_options="avahi doc python"
```

Comment:
I dont think we need those build options.

Reply:
Ok, removed them